python/foglamp/services/core/api/configuration.py

```diff
@@ -99,9 +103,7 @@
         category_desc = data.get('description')
         category_value = data.get('value')
 
-        should_keep_original_items = data.get('keep_original_items', False)
-        if not isinstance(should_keep_original_items, bool):
-            raise TypeError('keep_original_items should be boolean true | false')
+        should_keep_original_items = True if keep_original_items is not None and keep_original_items.lower() in ['true', '1', 1, 't'] else False
```

Comment:
It looks difficult to read.

It can be split in two or more lines

Reply:
@amarendra-dianomic 
https://github.com/foglamp/FogLAMP/pull/955/files#diff-8e6a39d1dfaaf562f1c60b2f201539bfR237
only `keep_original_items.lower() in ['true']`  is needed!